python/lsst/daf/butler/registry/_registry.py

```diff
@@ -614,7 +614,7 @@ def removeDatasetType(self, name: str | tuple[str, ...]) -> None:
 
         Parameters
         ----------
-        name : `str` or `tuple[str, ...]`
+        name : `str` or `tuple` [`str`]
```

Comment:
The ellipsis is important. It says a tuple with any number of strings, versus a tuple of a single string.

Reply:
Except that the "tuple" is logically being used as a list, not as a tuple, and [the convention for lists](https://developer.lsst.io/python/numpydoc.html#sequence-types) is pretty clear.

(IMO this method should take any iterable, but that's a code issue, not a documentation formatting issue.)